### PR TITLE
Onboard Grafana-Operator

### PR DESCRIPTION
**Pull request by hubeadmin**
Onboarding https://github.com/integr8ly/grafana-operator

### CODE REVIEW

**Comment by HubertStefanski** [comment]
@mikenairn @pb82 @david-martin mind having a look?

File: ci-operator/config/integr8ly/grafana-operator/integr8ly-grafana-operator-master.yaml
Comment (mikenairn) [inline ci-operator/config/integr8ly/grafana-operator/integr8ly-grafana-operator-master.yaml:5]: Any reason to use this base image? We use ubi for everything else

File: ci-operator/config/integr8ly/grafana-operator/OWNERS
Comment (mikenairn) [inline ci-operator/config/integr8ly/grafana-operator/OWNERS:7]: I think you have realised now, but these all need to be members of the OpenShift GH org. However, it will get synced from the repo once it's merged, so make sure you have an OWNERS file in there with the correct people listed as approvers.

File: ci-operator/config/integr8ly/grafana-operator/integr8ly-grafana-operator-master.yaml
Comment (pb82) [reply ci-operator/config/integr8ly/grafana-operator/integr8ly-grafana-operator-master.yaml:5]: No real reason other than the operator being a upstream focused project. But UBI would work for that too I guess. Would you recommend to change the base image @mikenairn ?

File: core-services/prow/02_config/_plugins.yaml
Comment (mikenairn) [inline core-services/prow/02_config/_plugins.yaml:3511]: We removed this plugin (blunderbuss) for other integr8ly repos, up to you if you want to keep it though.

Comment (HubertStefanski) [reply core-services/prow/02_config/_plugins.yaml:3511]: I'll remove it here as well, we'd probably end up ignoring the blunderbuss requests anyway since it's either me or Peter reviewing

File: ci-operator/config/integr8ly/grafana-operator/integr8ly-grafana-operator-master.yaml
Comment (mikenairn) [inline ci-operator/config/integr8ly/grafana-operator/integr8ly-grafana-operator-master.yaml:10]:
```
build_root:
  project_image:
    dockerfile_path: openshift-ci/Dockerfile.tools
```

**Comment by HubertStefanski** [comment]
@hongkailiu @droslean Mind having a look?

**Review by petr-muller — APPROVED** [review]

**Pull request merged**
